docs/getting-started/flashing.md

```diff
@@ -0,0 +1,123 @@
+# Flashing Zarhus OS
+
+There are many ways an OS can be flashed and booted on embedded platform, but
+the most common one is using an SD card. This guide will describe SD card
+flashing only, because all other ways to flash the image are highly
+target-specific and are described [per-target](../supported-targets/targets.md).
+
+## Flashing SD card
+
+This section demonstrates how to flash a Zarhus OS image on the SD card.
+
+### Prerequisites
+
+* Linux PC (tested on `Ubuntu 20.04 LTS`);
+* [bmaptool](https://source.tizen.org/documentation/reference/bmaptool)
+  installed:
+
+    ```bash
+    sudo apt install bmap-tools
+    ```
+
+* Zarhus OS image built according to [build guide](./building.md).
+
+!!! note
+
+    You can also use `bmap-tools` [from
+    GitHub](https://github.com/intel/bmap-tools) if it is not available in your
+    distro.
+
+### Flashing
+
+Find out your device name:
+
+```shell
+$ fdisk -l
```

Comment:
Maybe use `lsblk` as it's output is more concise and on my PC I need to use `sudo` with `fdisk` (unlike with `lsblk`). You also use `lsblk` in `### Verification`

Reply:
Changed here eaa2ab81ced422f758e8d85075b4cceb2aaad608.